x/stakeibc/keeper/icacallbacks_claim_test.go

```diff
@@ -71,6 +125,30 @@ func (s *KeeperTestSuite) TestClaimCallback_Successful() {
 
 	_, found := s.App.RecordsKeeper.GetUserRedemptionRecord(s.Ctx(), initialState.callbackArgs.UserRedemptionRecordId)
 	s.Require().False(found, "record has been deleted")
+
+	// fetch the epoch unbonding record
+	epochUnbondingRecord1, found := s.App.RecordsKeeper.GetEpochUnbondingRecord(s.Ctx(), tc.initialState.epochNumber)
+	s.Require().True(found, "epoch unbonding record found")
+	epochUnbondingRecord2, found := s.App.RecordsKeeper.GetEpochUnbondingRecord(s.Ctx(), tc.initialState.epochNumber+1)
+	s.Require().True(found, "epoch unbonding record found")
+
+	// fetch the hzus
+	hzu1 := epochUnbondingRecord1.HostZoneUnbondings[0]
+	hzu2 := epochUnbondingRecord1.HostZoneUnbondings[1]
+	hzu3 := epochUnbondingRecord2.HostZoneUnbondings[0]
+	hzu4 := epochUnbondingRecord2.HostZoneUnbondings[1]
+
+	// check that hzu1 has a decremented amount
+	s.Require().Equal(hzu1.NativeTokenAmount-tc.initialState.decrementAmount, hzu1.NativeTokenAmount, "hzu1 amount decremented")
```

Comment:
am I misunderstanding this test? is it just testing that `tc.initialSTate.decrementAmount` is 0? 

same question with the below tests too!

Reply:
Good catch! Decrement amount wasn't being set correctly. Updated